readme.md

```diff
@@ -161,7 +163,7 @@ If you would like git2consul to treat JSON documents in your repo as fully forme
 git2consul in expand_keys mode will generate the following KV:
 
 ```
-/expando_keys/root.json/first_level/second_level/third_level/you%20get%20the%20picture
+/expand_keys/root.json/first_level/second_level/third_level/you%20get%20the%20picture
```

Comment:
I think that since the repository is named 'expando_keys' we want to keep the 'o' here, right?